ThirdPartyNotices.txt

```diff
@@ -30,6 +30,7 @@ expressly granted herein, whether by implication, estoppel or otherwise.
     getmac: https://github.com/bevry/getmac
     graceful-fs: https://github.com/isaacs/node-graceful-fs
     gridstack: https://github.com/gridstack/gridstack.js
+		html-to-image: https://github.com/bubkoo/html-to-image
```

Comment:
fix spacing

Reply:
Thanks for the feedback, @Charles-Gagnon. I fixed the permissions on those files and moved the other core components into their own PRs.